functionality.js

```diff
@@ -4,13 +4,18 @@ import {
 
 import { createModal, projects } from './popupwindow.js';
 
+import {
+  isItUppercase, email, form, error,
+} from './validation.js';
+
```

Comment:
- [ ] Good job Implementing the form validations, However it looks like you did not select the form correctly and it is showing this error in browser console :point_down: 
![image](https://user-images.githubusercontent.com/97521557/204077186-03ddfb33-c3cd-46ba-930d-4dced354b42b.png)

Please fix your form as form = document.querySelector('form');

Also Please fix your error componenet as is is currently null, you do not have any element with the class of error in your Html, you can add a `<span class=error><span>` below the email input to display the error text 
![image](https://user-images.githubusercontent.com/97521557/204077300-4a1636e3-c550-4e7c-bd03-5aad2cd6fa9e.png)